### DISCUSSION TITLE
OrchardCore1.0.0-rc2-16238 with bootstrap 4.6.0 not available for Azure pipeline

### DISCUSSION THREAD

**Discussion by edmondlui-optionation** [OrchardCMS/OrchardCore#14084] [Q&A]
I have a production website built with OrchardCore 1.0.0-rc2-16238 and Bootstrap 4.6.0, which was created 2 years ago using Azure Pipeline.

Last week, we needed to make a small change, and we discovered that the 1.0.0-rc2 version is no longer available in the NuGet package manager with Azure Pipeline. As a result, we were forced to upgrade to version 1.0.0.

The issue we encountered is that OrchardCore 1.0.0 uses Bootstrap 5.0.2, which is different from the Bootstrap 4.6.0 used in OrchardCore 1.0.0-rc2.

Is it possible to include OrchardCore 1.0.0-rc2 in the NuGet manager or find a way to force OrchardCore 1.0.0 to use Bootstrap 4.6.0?

Please advise. Thanks a lot!

![image](https://github.com/OrchardCMS/OrchardCore/assets/109190080/96d1b8de-c457-4db9-9965-a235b1ff5b3c)

From Edmond

**Comment by BenedekFarkas** [comment]
Hi Edmond!

The way I see it, your problem has two distinct steps to it.

---

1. Get ahold of the packages of your desired version:

Do you have someone in your team that actually has those packages on their machine in the NuGet cache? If yes, then you're in a good position, so skip to the second point. :)

The oldest version stored on Cloudsmith at the moment is 1.6.0-preview-17265 (and they are constantly rotating out), so that won't help unfortunately. Older versions were on MyGet until some time ago, IIRC.
There might be a slim chance that someone knows roughly which commit hash your desired version is built from. Or you can just find the latest commit in the repo that uses Bootstrap 4.6.0. If that's the case, then rebuilding those packages is quite simple (it's slightly more complicated than just running `nuget publish`).
Alternative is upgrade to the closest known tag, but in both cases, you might face breaking changes.

---

2. Make those packages available in Azure Pipelines when you're building your solution.

If you have your own package source (IDK, maybe Azure Pipelines has a simple way to set one up), then publish the package there and you're good to go, but you'll probably need to add that package source in the repo or during the restore procedure in the pipeline.
Probably the quickest (but not so nice) way is to commit the packages to your repo and add that relative path to nuget.config.

**Reply by Piedone** [reply]
Also, if you want a quick and dirty, and perhaps working (but only perhaps) approach, you can try downgrading to [`1.0.0-rc2-13450`](https://www.nuget.org/packages/OrchardCore/1.0.0-rc2-13450) which is the closest to the version in question and still available on NuGet.
